webapp/src/ts/services/enketo.service.ts

```diff
@@ -632,6 +632,8 @@ export class EnketoFormContext {
   isFormInModal?: boolean;
   userContact?: Nullable<Person.v1.Person>;
   contactSummary?: Record<string, any>;
+  userContactId?: string;
+  userFacilityId?: string;
```

Comment:
Double checking: is this just one facility or can it be many?

Reply:
it can be many. as you do :D